teleband/assignments/api/views.py

```diff
@@ -101,7 +101,7 @@ def list(self, request, *args, **kwargs):
         }
 
         # FIXME: this should respect order from server/pieceplan and mayeb do this as a backup?
-        orderFromActivityType = lambda a: ordering[a['activity_type_name']]
+        orderFromActivityType = lambda a: ordering[a['activity_type_name'].split()[0]]
```

Comment:
This is from one of my previous PRs. This is what the frontend used to do I just forgot to move the split part over.